[INFO   ] Logger: Record log in /storage/emulated/0/1 projetosPy/SpaceInvader/.kivy/logs/kivy_20-08-03_31.txt
[INFO   ] Kivy: v1.11.1
[INFO   ] Kivy: Installed at "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.8.3 (default, Jun 15 2020, 02:19:10) 
[GCC 9.3.0]
[INFO   ] Python: Interpreter at "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/bin/python3"
[INFO   ] Factory: 184 symbols loaded
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil, img_gif (img_ffpyplayer ignored)
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/1 projetosPy/SpaceInvader/myApp.py", line 60, in <module>
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/app.py", line 828, in run
[WARNING] stderr:     self.load_kv(filename=self.kv_file)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/app.py", line 599, in load_kv
[WARNING] stderr:     root = Builder.load_file(rfilename)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 301, in load_file
[WARNING] stderr:     return self.load_string(data, **kwargs)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/builder.py", line 368, in load_string
[WARNING] stderr:     parser = Parser(content=string, filename=fn)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 401, in __init__
[WARNING] stderr:     self.parse(content)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 510, in parse
[WARNING] stderr:     objects, remaining_lines = self.parse_level(0, lines)
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 613, in parse_level
[WARNING] stderr:     _objects, _lines = self.parse_level(
[WARNING] stderr:   File "/data/data/ru.iiec.pydroid3/files/arm-linux-androideabi/lib/python3.8/site-packages/kivy/lang/parser.py", line 583, in parse_level
[WARNING] stderr:     raise ParserException(self, ln,
[WARNING] stderr: kivy.lang.parser.ParserException: Parser: File "/storage/emulated/0/1 projetosPy/SpaceInvader/myapp.kv", line 26:
[WARNING] stderr: ...
[WARNING] stderr:      24:		size: 50, 50
[WARNING] stderr:      25:		pos: self.pos
[WARNING] stderr: >>   26:	size_hint:None, None
[WARNING] stderr:      27:<Game>:
[WARNING] stderr:      28:	FloatLayout:
[WARNING] stderr: ...
[WARNING] stderr: Invalid data after declaration
